lib/deadletterProcessors/reenqueueByName.js

```diff
@@ -0,0 +1,12 @@
+const deadletterActions = require('./../deadletterActions');
+const Q = require('q');
+
+module.exports = function(event, message) {
+  const content = JSON.parse(message.content);
```

Comment:
`parse()` will throw a cheeky error if the message content isn't valid JSON, will this be handled gracefully by the library?

Reply:
in this context iirc the message is guaranteed to be valid json, also this would just cause our commandline app to crash